functions/__sf_section_elixir.fish

```diff
@@ -0,0 +1,57 @@
+#
+# Elixir
+#
+# A dynamic, reflective, object-oriented, general-purpose programming language.
+# Link: https://www.elixir-lang.org/
+
+function __sf_section_elixir -d "Show current version of Elixir"
+	# ------------------------------------------------------------------------------
+	# Configuration
+	# ------------------------------------------------------------------------------
+
+	__sf_util_set_default SPACEFISH_ELIXIR_SHOW true
+	__sf_util_set_default SPACEFISH_ELIXIR_PREFIX $SPACEFISH_PROMPT_DEFAULT_PREFIX
+	__sf_util_set_default SPACEFISH_ELIXIR_SUFFIX $SPACEFISH_PROMPT_DEFAULT_SUFFIX
+	__sf_util_set_default SPACEFISH_ELIXIR_SYMBOL "💧 "
+	__sf_util_set_default SPACEFISH_ELIXIR_DEFAULT_VERSION $SPACEFISH_ELIXIR_DEFAULT_VERSION
+	__sf_util_set_default SPACEFISH_ELIXIR_COLOR magenta
+
+	# ------------------------------------------------------------------------------
+	# Section
+	# ------------------------------------------------------------------------------
+
+	# Check if that user wants to show elixir version
+	[ $SPACEFISH_ELIXIR_SHOW = false ]; and return
+
+	# Show versions only for Elixir-specific folders
+	if not test -f mix.exs \
+		-o (count *.ex) -gt 0 \
+		-o (count *.exs) -gt 0
+		return
+	end
+
+	set -l elixir_version
+
+	if type -q kiex
+		set elixir_version $ELIXIR_VERSION
```

Comment:
This is probably the source of the test failures, because you’ve mocked `elixir -v`, but not set a mock value for `$ELIXIR_VERSION` or a mock for `exenv version-name`. I don’t know the specifics for Travis, but a different CI provider I use has a default version of Elixir installed using `kiex`.